cfspopcon/unit_handling/default_units.py

```diff
@@ -116,18 +132,23 @@
     spitzer_resistivity="m * ohm",
     summed_impurity_density="n19",
     surface_area="m**2",
+    surface_inductance_coefficients=None,
     target_electron_density="n19",
     target_electron_flux="m**-2 s**-1",
     target_electron_temp="eV",
     target_q_parallel="GW / m**2",
     tau_e_scaling_uses_P_in=None,
     temperature_peaking="",
     toroidal_flux_expansion="",
+    total_flux_available_from_CS="weber",
     trapped_particle_fraction="",
     triangularity_psi95="",
     triangularity_ratio_sep_to_psi95="",
     two_point_model_method=None,
     upstream_electron_temp="eV",
+    vertical_field_mutual_inductance="",
```

Comment:
Should this be `henry`?

Reply:
No. This is a unitless "mutual inductance" from one of the papers